lib/traject/psulib_config.rb

```diff
@@ -359,6 +359,143 @@
 end
 
 ## Notes fields
+#
+# 500 - Note
+to_field 'general_note_ssm', extract_marc('5003a:5903a')
+
+# 501 - "With"
+to_field 'with_note_ssm', extract_marc('501a')
+
+# 502 - Dissertation Note
+to_field 'dissertation_note_ssm', extract_marc('502abcdgo')
+
+# 504 - Bibliography Note
+to_field 'bibliography_note_ssm', extract_marc('504ab')
+
+# 505 - Table of Contents
+to_field 'toc_ssim', extract_marc('505agrt')
+
+# 506 - Restrictions on Access
+to_field 'restrictions_access_note_ssm', extract_marc('506abcdefu')
+
+# 507 - scale note for graphic material
+to_field 'scale_graphic_material_note_ssm', extract_marc('507ab')
+
+# 508 - Creation/Production Credits Note
+to_field 'creation_production_credits_ssm', extract_marc('508a')
+
+# 510 - Citation/References Note
+to_field 'citation_references_note_ssm', extract_marc('510| *|3abcux')
+to_field 'indexed_by_note_ssm', extract_marc('510|0*|3abcux:510|1*|3abcux')
+to_field 'selectively_indexed_by_note_ssm', extract_marc('510|2*|3abcux')
+to_field 'references_note_ssm', extract_marc('510|3*|3abcux:510|4*|3abcux')
+
+# 511 - Participant/Performer Note
+to_field 'participant_performer_ssm', extract_marc('511a')
+
+# 513 - Type of Report and Period Covered Note
+to_field 'type_report_period_note_ssm', extract_marc('513ab')
+
+# 515 - Special Numbering
+to_field 'special_numbering_ssm', extract_marc('515a')
+
+# 516 - Type of File/Data
+to_field 'file_data_type_ssm', extract_marc('516a')
+
+# 518 - Data/Place of Event
+to_field 'date_place_event_note_ssm', extract_marc('518adop3')
+
+# 520 - Notes Summary
+to_field 'notes_summary_ssim', extract_marc('520ab')
+
+# 521 - Audience
+to_field 'audience_notes_ssm', extract_marc('521| *|3ab:521|8*|3ab:521|3*|3ab:521|4*|3ab')
```

Comment:
@cdmo space here as well: `521| *|3ab`, it is probably OK and I also wanted to see if it means anything?

Reply:
Yes, actually my example in the previous conversation is from `521`